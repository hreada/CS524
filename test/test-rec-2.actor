letactor(
  bind(test, rec(lambda(y, lambda(x, seq(send(x,5),become(y)))))),
  send(test, initactor))