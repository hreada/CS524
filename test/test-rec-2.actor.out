<T>
  <actors>
    <actor>
      <k> exec ( nil ) </k>
      <id> _10 </id>
    </actor>
    <actor>
      <k> ready ( lambda ( _5 , app ( app ( lambda ( _4 , app ( lambda ( y , lambda ( x , seq ( send ( x , 5 ) , become ( y ) ) ) ) , lambda ( _5 , app ( app ( _4 , _4 ) , _5 ) ) ) ) , lambda ( _4 , app ( lambda ( y , lambda ( x , seq ( send ( x , 5 ) , become ( y ) ) ) ) , lambda ( _5 , app ( app ( _4 , _4 ) , _5 ) ) ) ) ) , _5 ) ) ) </k>
      <id> _1 </id>
    </actor>
    <actor>
      <k> exec ( nil ) </k>
      <id> initactor </id>
    </actor>
  </actors>
  <messages>
    <message> msg ( initactor , 5 ) </message>
  </messages>
  <definedaddr> ListItem ( initactor ) ListItem ( _1 ) ListItem ( _10 ) </definedaddr>
</T>
